fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowAnalyzeJobStmt.java

```diff
@@ -80,13 +80,13 @@ public static List<String> showAnalyzeJobs(ConnectContext context,
 
             if (!analyzeJob.isAnalyzeAllTable()) {
                 String tableName = analyzeJob.getTableName();
-                Table table = GlobalStateMgr.getCurrentState().getMetadataMgr().getTable(analyzeJob.getCatalogName(),
-                        dbName, tableName);
+                BasicTable table = GlobalStateMgr.getCurrentState().getMetadataMgr().getBasicTable(
```

Comment:
This interface always returns a table, whether it exists or not

Reply:
we are doing it just to take table name and to check permissions
the table existed when we created a job

anyway happy to amend it if you say how